extensions/wikia/UserLogin/UserSignupSpecialController.class.php

```diff
@@ -239,15 +239,19 @@ public function signup() {
 			$ret = $signupForm->addNewAccount();
 		}
 
-		$this->result = ( $signupForm->msgType == 'error' ) ? $signupForm->msgType : 'ok';
-		$this->msg = $signupForm->msg;
-		$this->errParam = $signupForm->errParam;
-
 		// pass and ID of created account for FBConnect feature
 		if ( $ret instanceof User ) {
 			$this->userId = $ret->getId();
 			$this->userPage = $ret->getUserPage()->getFullUrl();
 		}
+
```

Comment:
We've been using ```setData``` or ```setValues``` whenever we update or create new controllers to make it more explicit that we're passing something to the reponse, e.g.:
```php
$this->response->setValues( [
    'result' => ( $signupForm->msgType == 'error' ) ? $signupForm->msgType : 'ok',
    'msg' => $signupForm->msg,
    'errParam' => $signupForm->errParam,
] );
```